modules/effects/arrow-effects-data/src/main/kotlin/arrow/effects/Resource.kt

```diff
@@ -15,7 +15,6 @@ import arrow.typeclasses.Semigroup
  * When a resource is created one can make use of [invoke] to run a computation with the resource. The finalizers are then
  *  guaranteed to run afterwards in reverse order of acquisition.
  **/
-@documented
```

Comment:
why can't this be documented?

Reply:
idk I am not familiar enough with those annotations. The build failed because of it tho.

Reply:
At least that is what I read from the error